core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -553,6 +558,7 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.submitAnswer = function(answer, interactionRulesService) {
+            $scope.pendingCardWasSeenBefore = false;
```

Comment:
Why do we set this here?

Reply:
I meant this as a way to reset pendingCardWasSeenBefore at each card as I think because of this, sometimes, Learn Again was shown permanently instead of Continue as a true value set before at some other card is never reset.

Reply:
But why was Learn Again set in those cases? That's the bug that needs fixing...

Reply:
(This function just gets called when a learner submits an answer. If the behaviour is wrong at this point, it would have been wrong in the moments before that.)

Reply:
It could be put in any function that is refreshed at each card, but the Continue button would be called only after the submitAnswer was called, that's why I put in this function, should I put in some other function that refreshes at every card?

Reply:
Yeah, I think so. That way, it's tied directly to the refresh event.

Reply:
Ok, will do that